internal/resources/metal/virtual_circuit/resource.go

```diff
@@ -111,12 +119,33 @@ func Resource() *schema.Resource {
 				Optional:     true,
 				RequiredWith: []string{"vrf_id"},
 				Description:  "The Metal IP address for the SVI (Switch Virtual Interface) of the VirtualCircuit. Will default to the first usable IP in the subnet.",
+				Computed:     true,
 			},
 			"customer_ip": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				RequiredWith: []string{"vrf_id"},
 				Description:  "The Customer IP address which the CSR switch will peer with. Will default to the other usable IP in the subnet.",
+				Computed:     true,
```

Comment:
Same as above, we only need one of `RequiredWith` or `Computed`.  Adopting `Computed` may require changes elsewhere in the resource.